src/wagtailmedia/templates/wagtailmedia/chooser/chooser-next.html

```diff
@@ -0,0 +1,90 @@
+                {% trans "Search" as search_text %}
+                {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='search' title=search_text %}
+                {% if uploadforms.audio %}
+                    {% trans "Upload Audio" as upload_audio_text %}
+                    {% if uploadforms.audio.errors and media_type == 'audio' %}
+                        {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='upload-audio' title=upload_audio_text errors_count=uploadforms.audio.errors|length %}
+                    {% else %}
+                        {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='upload-audio' title=upload_audio_text %}
+                    {% endif %}
+                {% endif %}
+                {% if uploadforms.video %}
+                    {% trans "Upload Video" as upload_video_text %}
+                    {% if uploadforms.video.errors and media_type == 'video' %}
+                        {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='upload-video' title=upload_video_text errors_count=uploadforms.video.errors|length %}
+                    {% else %}
+                        {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='upload-video' title=upload_video_text %}
+                    {% endif %}
+                {% endif %}
+            </div>
+        </div>
+    {% endif %}
+
+    <div class="tab-content nice-padding">
+        <section id="tab-search" class="w-tabs__panel" role="tabpanel" aria-labelledby="tab-label-search">
+            <form class="media-search search-bar" action="{% url 'wagtailmedia:chooser' %}" method="GET" novalidate>
+                <ul class="fields">
+                    {% for field in searchform %}
+                        {% include "wagtailadmin/shared/field_as_li.html" with field=field %}
+                    {% endfor %}
+                    {% if collections %}
+                        {% include "wagtailadmin/shared/collection_chooser.html" %}
+                    {% endif %}
+                    {% if popular_tags %}
+                        <li class="taglist">
+                            <h3>{% trans 'Popular tags' %}</h3>
+                            {% for tag in popular_tags %}
+                                <a class="suggested-tag tag" href="{% url 'wagtailmedia:index' %}?tag={{ tag.name|urlencode }}">{{ tag.name }}</a>
+                            {% endfor %}
+                        </li>
+                    {% endif %}
+                </ul>
+            </form>
+            <div id="search-results" class="listing media">
+                {% include "wagtailmedia/chooser/results.html" %}
+            </div>
+        </section>
+        {% if uploadforms %}
+            {% for form_type, uploadform in uploadforms.items %}
+            <section id="tab-upload-{{ form_type }}" class="w-tabs__panel" role="tabpanel" aria-labelledby="tab-label-upload-{{ form_type }}">
```

Comment:
```suggestion
            <section id="tab-upload-{{ form_type }}" class="w-tabs__panel" role="tabpanel" aria-labelledby="tab-label-upload-{{ form_type }}" hidden>
```
This is also analog to the image's/document's chooser and this was actually the cause for the blanks, noted in the above issue with `data-tabs-disable-url`. It also caused some blanks added to the bottom if the chooser was initially opened.
I think adding `data-tabs-disable-url` still seems better, so its similar to the other choosers, and also keeping the hash in the url makes no sense to me, because normally you will choose an upload tab, upload the file, which closes the chooser. Now the hash is still in the browsers location...